src/sbvr-api/compile-auth.ts

```diff
@@ -0,0 +1,51 @@
+import type { Filter, PinejsClientCore, Resource } from 'pinejs-client-core';
+
+const asPermission = (compiledFilter: string): string => {
+	return compiledFilter
+		.split('$filter=')[1]
+		.replace(/Auth\.canAccess\(\)/g, 'canAccess()');
+};
```

Comment:
You should be able to do something like:
``````suggestion
const asPermission = (url: string): string => {
	const queryParamsIndex = url.indexOf('?');
	if (queryParamsIndex === -1) {
		throw ...
	}
	const queryParams = new URLSearchParams(url.slice(queryParamsIndex));
	const compiledFilter = queryParams.get('$filter');
	if (compiledFilter == null) {
		throw ...
	}
	return compiledFilter.replace(/Auth\.canAccess\(\)/g, 'canAccess()');
};
```
to ensure proper parsing of it as query params and whatnot